masterlist.yaml

```
@@ -25514,3 +25514,28 @@ plugins:
         util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
         udr: 3
+  - name: 'SurWR.esp'
```

Sorry about that, the suggestion wasn't supposed to remove the newline as well. I forgot that GH's suggestions can be a bit finicky at times:

```suggestion

  - name: 'SurWR.esp'
```